hibernate-cache/src/main/resources/org/hibernate/cache/infinispan/builder/infinispan-configs-local.xml

```diff
@@ -17,20 +17,22 @@
 
       <!-- Default configuration is appropriate for entity/collection caching. -->
       <local-cache-configuration name="entity" simple-cache="true" statistics="false" statistics-available="false">
+         <locking isolation="READ_COMMITTED"/>
```

Comment:
I see that the query cache is also non-transactional, is the test overriding that? (probably it is running in both variants). In any case, it should not pollute the default configuration.

And breaking a test with RR sounds fishy, have you investigated?